patternlab/scripts/twig_functions.sh

```diff
@@ -25,7 +25,6 @@ then
   # Array of twig functions to copy, starting from the vendor directory
   twig_functions=(
     "drupal-pattern-lab/add-attributes-twig-extension/add_attributes.function.php"
-    "drupal-pattern-lab/bem-twig-extension/bem.function.php"
```

Comment:
This is the main thing that changed

Reply:
Ah, so just pulling this up out of the library code and into our code?

Reply:
I think we'll still potentially need it with the storybook/twig changes to interpolate the bem function in all of those files, but we should either include it as a dependency or not have some install script moving it around 😅